### PR TITLE
Add the first addons for home assistant

### PR DESCRIPTION
**Pull request by LukBelter**
First Version of BCO as addon for home assistant.

### CODE REVIEW

File: bco-core/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# BCO Core
+Runs the BCO Core Docker container.
+
+Healthcheck is still missing.
```
Comment (DivineThreepwood) [inline bco-core/README.md]:
```suggestion
```

File: bco-homeassistant/config.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - aarch64
+  - amd64
+startup: application
+ports:
+  13781/tcp: 13781
+boot: manual
+image: openbaseorg/bco-device-homeassistent
+environment:
+  BCO_OPTIONS: "--host local-bco-middleware --port 13780"
+# watchdog: tcp://[HOST]:13781
```
Comment (DivineThreepwood) [inline bco-homeassistant/config.yaml]:
```suggestion
```

File: bco-middleware/config.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+slug: "bco-middleware"
+arch:
+  - aarch64
+  - amd64
+  - armv7
+  - i386
+startup: services
+ports:
+  13780/tcp: 13780
+boot: manual
```
Comment (DivineThreepwood) [inline bco-middleware/config.yaml]: is there a auto boot?

File: bco-core/config.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+url: "https://www.openbase.org"
+version: dev
+slug: "bco-core"
+arch:
+  - aarch64
+  - amd64
+startup: application
+ports:
+  13781/tcp: 13781
+boot: manual
```
Comment (DivineThreepwood) [inline bco-core/config.yaml]: ?

File: bco-ui/config.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+url: "https://basecubeone.org/docs/user/howto/setup-docker#deploy-the-bco-webapp-docker"
+version: latest
+slug: "bco-ui"
+arch:
+  - aarch64
+  - amd64
+startup: application
+ports:
+  80/tcp: 80
+boot: manual
```
Comment (DivineThreepwood) [inline bco-ui/config.yaml]: ?

**Review by DivineThreepwood — CHANGES_REQUESTED** [review]

**Review by lhuxohl — APPROVED** [review]

**Review by DivineThreepwood — APPROVED** [review]

**Pull request merged**